Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:latest
+
+# Install dependencies
+RUN apt update
+RUN apt install -y build-essential curl git wget
+RUN apt install -y qemu-system-aarch64 dosfstools mtools
+
+# Install Rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
Maybe we should only install nightly in a future PR. Also installing the required toolchains manually would help with caching.